FurEver/Pages/DogQuiz.cshtml.cs

```diff
@@ -0,0 +1,113 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace FurEver.Pages
+{
+    public class DogQuizModel : PageModel
+    {
+        public string QuizImageUrl { get; set; }
+        public List<string> BreedOptions { get; set; }
+        public string AnswerResult { get; set; }
+        public bool QuizFinished { get; set; } = false;
+
+        [TempData]
+        public int Score { get; set; }
+
+        [TempData]
+        public int CurrentQuestionNumber { get; set; } = 1;
+
+        [TempData]
+        public string CorrectBreed { get; set; }  // Store correct breed in TempData
+
+        public readonly int TotalQuestions = 5;
```

Comment:
# Technical Change #3:
## Refactoring hardcoded values into constants or read-only values
This technical change reduces technical debt by improving code maintainability, readability, and flexibility. Refactoring hardcoded values into constant or read-only variables clarifies to developers that the value is intended to remain unchanged during the program's execution. This makes the code more self-documenting, reducing the likelihood of accidental modifications and preventing bugs that may arise from unintended changes. Additionally, by centralizing these values as constants or readonly fields, future modifications become simpler, since developers can update the value in one place rather than searching for multiple hardcoded instances throughout the code. This enhances the code's flexibility and adaptability, making it easier to accommodate future requirements. 
Link to References: https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/constants and https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/readonly

Reply:
Good idea.
Constants are usually named with UPPER_CASE_WITH_HYPHENS.  Do this when making a constant.